src/homepageExperience/components/steps/python/WriteData.tsx

```diff
@@ -116,7 +116,10 @@ for value in range(5):
         Once you write this data, you’ll begin to see the confirmation below
       </p>
 
-      <Panel backgroundColor={InfluxColors.Grey15}>
+      <Panel
+        backgroundColor={InfluxColors.Grey15}
+        style={{marginBottom: '24px'}}
```

Comment:
I might move this style outside the component, since the `marginBottom` style [could cause re-renders ](https://www.aaronbuxbaum.com/how-object-literals-hurt-performance/)